framework/components/ema-rating-form/ema-rating-form.twig

```diff
@@ -9,19 +10,25 @@
 
 {# Print result #}
 <form class="{{_css_class}}">
-  <div class="ecl-u-d-flex ecl-u-flex-wrap" aria-controls="ema-rating-form__review-1">
-    <div class="ecl-u-mr-l ecl-u-mb-s">
-      {% include '@ec-europa/ema-ratings' with _rating %}
+  <div class="ecl-u-d-flex ecl-u-flex-wrap" id={{ _selector }} aria-controls={{ _selector ~ "-block" }}>
+    {% for rating in ratings %}
+      <div class="ecl-u-mr-l ecl-u-mb-s">
+        {% include '@ec-europa/ema-ratings' with rating %}
+      </div>
+    {% endfor %}
+    <div class="ecl-u-d-flex ecl-u-align-items-center">
+      <a class="ecl-link--ema-default ecl-u-mb-s" href="#">View all 12 ratings</a>
     </div>
-
-    {% block extra_content extra_content %}
   </div>
-  <div id="ema-rating-form__review-1" class="ema-rating-form__review ecl-u-flex-wrap ecl-u-flex-md-nowrap ecl-u-align-items-end" aria-hidden="true">
-    {% include '@ec-europa/ecl-forms-textareas' with { rows: 3, extraClass: 'ecl-u-mr-md-s' } %}
-    {% include '@ec-europa/ema-buttons' with {
-      extra_classes: 'ecl-button--ema-primary ecl-u-mt-xxxs ema-u-width-100 ema-u-width-md-auto',
-      label: 'Submit'
-    } %}
+  <div id={{ _selector ~ "-block" }} class="ema-rating-form__review ecl-u-flex-column" aria-hidden="true">
+    <p class="ecl-u-ma-none ecl-u-fs-s"><strong>{{ _message }}</strong></p>
```

Comment:
Display of the message is a little too aggressive to my mind. 
I understand that you mimic the display of existing rating block, but it would probably be smoother if we didn't put it in bold (and with default font size)